client/components/PubReleaseDialog/PubReleaseDialog.tsx

```diff
@@ -100,7 +98,7 @@ const PubReleaseDialog = (props: Props) => {
 			<div>
 				<p className="text-info">{label}</p>
 				<ControlGroup className="url-select">
-					<InputGroup className="display-url" value={url} fill small />
+					<InputGroup className="display-url" value={url} fill small disabled />
```

Comment:
Thanks for the attention to detail here. An alternative fix that doesn't need any CSS changes is:
```tsx
onChange={() => {}}
```
presented for your consideration.